engine/schema/src/main/resources/META-INF/db/schema-41310to41400.sql

```diff
@@ -53,33 +53,6 @@ ALTER TABLE `cloud`.`vm_instance` ADD COLUMN `backup_offering_id` bigint unsigne
 ALTER TABLE `cloud`.`vm_instance` ADD COLUMN `backup_external_id` varchar(255) DEFAULT NULL COMMENT 'ID of external backup job or container if any';
 ALTER TABLE `cloud`.`vm_instance` ADD COLUMN `backup_volumes` text DEFAULT NULL COMMENT 'details of backedup volumes';
 
-ALTER TABLE `cloud`.`image_store` ADD COLUMN `readonly` boolean DEFAULT false COMMENT 'defines status of image store';
```

Comment:
why is this moved to another upgrade script? it makes it harder to compare and would imply it will not be needed/used in propriatory 4.13 releases anymore

Reply:
@DaanHoogland  This is related to PR#4053 which merged recently to master where the schema changes were made in old schema file. 
https://github.com/apache/cloudstack/pull/4053/files#r490066335